<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradeDialog</class>
 <widget class="QDialog" name="GradeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成绩信息</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="add">
      <property name="text">
       <string>增</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="delete_">
      <property name="text">
       <string>删</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="change">
      <property name="text">
       <string>改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="search">
      <property name="text">
       <string>查</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>实验序号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ExpNo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>    成   绩</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Score"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>     学  号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StuNo"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>361</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
